### PR TITLE
Fix the build by bumping to crate-docs 1.0.0

### PR DESCRIPTION
**Pull request by amotl**
Hi there,

this patch fixes https://github.com/crate/crate-docs/issues/65 reported by @msbt. You can either choose this one, or bring in #245 instead. Or do both, one after the other.

With kind regards,
Andreas.

### CODE REVIEW

**Comment by norosa** [comment]
@amotl can you bring in my changes to `docs/Makefile` (https://github.com/crate/crate-docs-theme/pull/245/files). this is required for a proper upgrade. see the release notes: https://github.com/crate/crate-docs/releases/tag/1.0.0

I'm happy to then merge this pr and drop the related commit from https://github.com/crate/crate-docs-theme/pull/245

**Comment by norosa** [comment]
side note: I am not clear on how this fixes https://github.com/crate/crate-docs/issues/65. or rather, I am not clear on what causes https://github.com/crate/crate-docs/issues/65. can you elaborate?

**Comment by amotl** [comment]
> I am not clear on how this fixes crate/crate-docs#65. or rather, I am not clear on what causes crate/crate-docs#65. can you elaborate?

I believe the reason is that crate-docs 0.4.0 explicitly tests for python3.7. However, I don't know exactly. Let's just go ahead.

**Review by norosa — APPROVED** [review]
I think it's a bit unconventional to do the Makefile change in a separate pr (https://github.com/crate/crate-docs-theme/pull/247) but the end result will be the same so I don't mind. let's go ahead

File: docs/index.rst [hunk 1]
```diff
@@ -125,5 +125,5 @@ Fifth-level heading
 This is the deepest heading level we support.
 
 
-.. _developer guide: https://github.com/crate/crate-docs-theme/blob/master/DEVELOP.rst#make-changes
+.. _developer guide: https://github.com/crate/crate-docs-theme/blob/master/DEVELOP.rst#user-content-make-changes
```
Comment (amotl) [inline docs/index.rst:128]: Attaching to this, I just created https://github.com/sphinx-doc/sphinx/issues/9016.

**Pull request merged**